imap/http_caldav_sched.c

```diff
@@ -348,7 +348,13 @@ static int imip_send_sendmail(const char *userid, icalcomponent *ical, const cha
 
     buf_printf(&msgbuf, "Subject: %s: ", filename);
     if (summary) {
-        char *mimehdr = charset_encode_mimeheader(summary, 0, 0);
+        size_t len = strlen(summary);
+
+        /* Trim any trailing CR and/or LF */
+        while (strchr("\r\n", summary[len-1])) len--;
+
+        /* force=1 to encode any non-folding LF */
+        char *mimehdr = charset_encode_mimeheader(summary, len, 1 /* force */);
```

Comment:
doesn't force_quote cause all string values to be encoded, even sane short ASCII-only ones?

Reply:
Yes.  It appears as if non-folding LF don't get captured in our checks, which why I used force.  I didn't want to change that check for this specialty case